### ISSUE TITLE
error_permission error on ANDROID

### ISSUE THREAD

**Issue by ascocoli** [csdcorp/speech_to_text#387]
#301 Hello ! I hope your are so fine. I need your help. I cloned the package example and i got this error: error_permission
I used all permissions like recommanded in my androidManifest. 

`<uses-permission android:name="android.permission.INTERNET"/>
    <uses-permission android:name="android.permission.BLUETOOTH"/>
    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
    <uses-permission android:name="android.permission.BLUETOOTH_CONNECT"/>
    <uses-permission android:name="android.permission.RECORD_AUDIO" />
    <queries>
        <intent>
            <action android:name="android.speech.RecognitionService" />
        </intent>
    </queries>`
[Projet github repository](https://github.com/ascocoli/gpt)
It's work very fine on web but not for android. 
thank to help me !
this is my screenshot: 
![Screenshot_20230427-102829](https://user-images.githubusercontent.com/69978395/234840202-ac6b5d22-c51d-404a-81ce-62c26763d42d.jpg)

**Comment by sowens-csd** [comment]
Does the example app do the same? Are you trying it on a simulator or a real device?

**Comment by ascocoli** [comment]
I tried both them . Android 11 physical device and emulator also. But no success

**Comment by ascocoli** [comment]
> 

Yes it's the app example. I tried both them . Android 11 physical device and emulator also. But no success

**Comment by sowens-csd** [comment]
Can you retest with 6.3.0 if you have a chance?

**Comment by georges-ph** [comment]
@sowens-csd I'm using the version `6.3.0` and I'm having the same problem

**Comment by sowens-csd** [comment]
What version of Android are you using? What kind of Android device? Physical device or emulator?

**Comment by georges-ph** [comment]
It's a Samsung Galaxy E5 running Lineage OS 14.1
